db/migrate/20220820130343_add_user_ref_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
+  end
+end
```

Comment:
- [ ] The reference is not directly to user, it is instead author to table users. Kindly see the below example on how to make such migration for adding a reference with a different name than the reference model class name and make changes in your database. 🙏

![image](https://user-images.githubusercontent.com/77190203/183665204-85d7899a-ec56-4c94-9d6e-4325d1a89de4.png)

Reply:
Alright.
Thank you @medaminedev66 for the hint.